app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -308,6 +299,7 @@ class SuggestedEditsTasksFragment : Fragment() {
     }
 
     private fun showOneTimeSequentialUserStatsTooltips() {
+        suggestedEditsScrollView.post(Runnable { suggestedEditsScrollView.fullScroll(View.FOCUS_UP) })
```

Comment:
Still seeing the tooltips jumping from outside of the card view to the position it should be in, but I guess it should be fine. @dbrant Any thoughts?

Reply:
I added a half second delay before starting the tooltip relay. Looks much better to me. Lmk your thoughts @cooltey and @dbrant